main.js

```diff
@@ -0,0 +1,161 @@
+
+// Task #1
+
+// function createArray() {
+//     let userArray = [];

```

Comment:
Pushing elements in array != changing a value of variable

```suggestion
//     const userArray = [];

```